### ISSUE TITLE
[Question] State of group chat

### ISSUE THREAD

**Issue by mimi89999** [zom/Zom-Android-XMPP#102]
From the description it is not clear if group chats are encrypted or not. What is the current situation? I know about the problems with Axolotl/OMEMO. Under an issue on Gitlab you stated that "We had an OTR group encryption working on Android". So are encrypted group chats possible with OTR?

**Comment by n8fr8** [comment]
Yes, using "pair wise" encrypted OTR sessions between each participant in the room. This is implemented on Zom Android, but disabled, since we don't have it on ChatSecure/Zom-iOS and since other XMPP clients don't support it.

We can update the app info to make sure that it is clear we are only using TLS with group chats. In addition, the chat UI will show a shield icon to indicate if a message is encrypted or not, so in group chats this icon is not present.

**Comment by mimi89999** [comment]
Please update the app info. I understood, that everything is e2e OTR encrypted.

**Comment by n8fr8** [comment]
Encrypted group chat is a very complicated thing to do and no one has
yet to do it in an open, interoperable standards based way. There are
plenty of proprietary solutions but we aren't interested in that.

**Comment by n8fr8** [comment]
We hope to support Olm soon: https://matrix.org/git/olm
and are also working with the mpotr / np1sec effort: https://github.com/equalitie/np1sec

**Comment by mimi89999** [comment]
> using "pair wise" encrypted OTR sessions between each participant in the room

That doesn't sound very good... How 'stable' is it?

**Comment by n8fr8** [comment]
This is the exact same technique used by Signal. Go read Moxie's blog
posts on the subject. It works just fine.

**Comment by mimi89999** [comment]
They aren't using OTR. They are using Axolotl...

**Comment by mimi89999** [comment]
Link?

**Comment by n8fr8** [comment]
I meant the "pair wise" aspect and not the encryption protocol specifically.

https://whispersystems.org/blog/private-groups/

"Given the ease with which TextSecure can send encrypted pairwise messages with forward secrecy and deniability, we decided to implement group messaging by employing pairwise messaging. When sending a message to a group, we simply deliver a pairwise encrypted message to each member of the group."
